examples/chart-chart/index.html

```diff
@@ -0,0 +1,33 @@
+<html>
+
+<head>
+    <script src="../../node_modules/seedrandom/seedrandom.js"></script>
+    <script>Math.seedrandom('a22ebc7c488a3a47');</script>
+    <script src="../../node_modules/d3/dist/d3.js"></script>
+    <script src="../../packages/d3fc/build/d3fc.js"></script>
+    <style>
```

Comment:
We're missing the default style rules, what's setting the dimensions of the charts?